controllers/src/main/scala/cromwell/pipeline/controller/ProjectSearchController.scala

```diff
@@ -0,0 +1,33 @@
+package cromwell.pipeline.controller
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.{ ExceptionHandler, Route }
+import cromwell.pipeline.controller.ProjectSearchController.projectSearchExceptionHandler
+import cromwell.pipeline.datastorage.dto._
+import cromwell.pipeline.datastorage.dto.auth.AccessTokenContent
+import cromwell.pipeline.service.ProjectSearchService
+import de.heikoseeberger.akkahttpplayjson.PlayJsonSupport._
+
+class ProjectSearchController(projectSearchService: ProjectSearchService) {
+
+  private def searchProjects(implicit accessToken: AccessTokenContent): Route = post {
+    entity(as[ProjectSearchRequest]) { request =>
```

Comment:
You also need to add `pathEnd` here. Otherwise it will work with unwanted paths like `search/foo/bar/buzz` which might lead to conflicts

Reply:
Fixed